### PR TITLE
created initial ui for terminal

### PR DESCRIPTION
**Pull request by IonLi09**
Wrote html, css, and js for a ui. run it on a local webserver

### CODE REVIEW

**Review by ianwood103 — COMMENTED** [review]
this SUCKS!!!!!

looks great

File: ground-station-openmct/terminal.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        color: #fff;
+        outline: none;
+        caret-color: #fff;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="terminal"></div>
+    <input id="input" placeholder="$" autocomplete="off" />
+    <script>
```
Comment (VanL15) [inline ground-station-openmct/terminal.html:39]: Put the JavaScript code in a different file, and then call the js code in the script tag.

Ex: <script src="terminal-ui.js"></script>

**Review by VanL15 — CHANGES_REQUESTED** [review]
see comment

**Review by VanL15 — APPROVED** [review]
lgtm

**Pull request merged**